### ISSUE TITLE
Deploy app for mid-project review

### ISSUE THREAD

**Issue by vanFrZy** [Bragi3000/Bragi3000#48]
We should deploy once the tictactoe is merged for the mid-project review

**Comment by nikolim** [comment]
I could write a github action for this

**Comment by Atema** [comment]
The firebase CLI can even do this automatically, although I'm not sure how well it works

**Comment by nikolim** [comment]
Yeah yeah tested and it gives the option to add a Github Action, would be nice to automatically redeploy the main branch

**Comment by Atema** [comment]
I've done a basic deploy to https://bragi-3000.web.app and added the config files in #49, so if you want to take a look at github actions etc. feel free :)

**Comment by Atema** [comment]
(When deploying, make sure to use the right `Config/spotify.js`, with the right callback url and a spofity project that allows the callbakc url)

**Comment by Atema** [comment]
Also, problem that I thought about is that the reviewers won't be able to use our deployed version if their Spotify username is not in our spotify developer project

**Comment by vanFrZy** [comment]
> Also, a problem that I thought about is that the reviewers won't be able to use our deployed version if their Spotify username is not in our spotify developer project

True, maybe we should tell them to email us their usernames?

**Comment by nikolim** [comment]
Ok this was fast, good work

**Comment by Atema** [comment]
> > Also, a problem that I thought about is that the reviewers won't be able to use our deployed version if their Spotify username is not in our spotify developer project
> 
> True, maybe we should tell them to email us their usernames?

Yeah, that's the only way, I guess

**Comment by nikolim** [comment]
@Atema the Github Action might be a bit more complicated, since we have to replace the config files

**Comment by Atema** [comment]
> @Atema the Github Action might be a bit more complicated, since we have to replace the config files

Could still be relatively easy using something like action secrets, I think. But not really necessary anyhow
